app/javascript/src/Policies/components/CustomPolicy.jsx

```diff
@@ -44,9 +44,18 @@ function CSRFToken ({win = window}: {win?: any}) {
   )
 }
 
-function CustomPolicyForm ({policy, onChange}: {policy: Policy, onChange: OnChange}) {
+function CustomPolicyForm ({policy, onChange, win = window}: {policy: Policy, onChange: OnChange, win?: Window}) {
   const isNewPolicy = !(policy.id && policy.id !== 0)
```

Comment:
To fix codeclimate erorrs maybe
```suggestion
  const isNewPolicy = !policy.id
```